Makefile

```diff
@@ -124,6 +124,9 @@ compile-requirements: $(COMMON_CONSTRAINTS_TXT) ## Re-compile *.in requirements
 	sed 's/Django<2.3//g' requirements/common_constraints.txt > requirements/common_constraints.tmp
 	mv requirements/common_constraints.tmp requirements/common_constraints.txt
 
+	pip install -q pip-tools
+	pip-compile --allow-unsafe --upgrade -o requirements/edx/pip.txt requirements/edx/pip.in
```

Comment:
Can't we just add the `pip` file in REQ_FILES [here](https://github.com/openedx/edx-platform/blob/cf1c43173b3d7a494b45743912fa636b7fe613b7/Makefile#L98) instead of adding the command separately...?

Reply:
We need an extra `--allow-unsafe` parameter since `pip-compile` by default doesn't allow these packages to be updated so we can't group this up with other paths.